components/AudioPlayer.module.css

```
@@ -250,96 +254,60 @@
 	}
 }
 
-
 /*================ Laptop ==================*/
 @media screen and (min-width: 1024px) {
+	.audio_player_section_container {
+		grid-template-columns: 50% 40%;
+		grid-template-rows: 65% 35%;
+		grid-template-areas:
+			'thumbnail description'
+			'player player';
+		gap: 3rem 0;
+	}
 
-	/* .audio_player_section_container {
-	  padding: 3%;
-	  } */
-  
-	  /* .flex_wrapper{
-		display: flex;
-		justify-content: space-evenly;
-		margin: 5%;
-	  } */
-  
-	  /* .image_wrapper{
-		width: 400px;
-		height: 400px;
-	  } */
-  
-	  /* .episode_info_section{
-		top: 0;
-		width: 400px;
-		height: 400px;
-		padding: 0;
-	  } */
-  
-	  /* .episode_title h3{
-		font-size: 35px;
-		padding: 5px;
-	  } */
-  
-	  /* .episode_title h4{
-		padding: 5px;
-		font-size: 20px;
-	  } */
-  
-	  /* .episode_details{
-		width: 100%;
-		height: 300px;
-	  } */
-  
-	  /* .audioPlayerPlaybutton{
-		flex: 1;
-		display: flex;
-		gap: 50px;
-		margin: 20px;
-	  } */
-  
-	  /* .audioPlayer {
-		display: flex;
-		justify-content: center;
-		gap: 30px;
-	  } */
-  
-	  /* .forwardBackward {
-		font-size: 24px;
-	  } */
-  
-	  /* .playPause {
-		width: 100px;
-		height: 100px;
-		font-size: 35px;
-	  } */
-  
-	  /* .play {
-		padding-left: 5px;
-	  } */
-  
-	  /* .currentTime,
-	  .duration {
-	  font-size: 24px;
-	  } */
-  
-	  /* .audioPlayerBar{
-		flex: 2.5;
-	  } */
-  
-	  /* .progressBar {
-		width: 80%;
-		height: 15px;
-	  } */
-  
-	  /* .progressBar::-webkit-slider-thumb {
-		height: 20px;
-		width: 20px;
-		bottom: 9px;
-	  } */
-  
+	.image_wrapper {
+		grid-area: thumbnail;
+		width: 70%;
+		height: 80%;
+	}
+
+	.episode_info_section {
+		grid-area: description;
+		margin-right: 10rem;
+	}
+
+	.episode_details {
+		padding: 3.5rem 0;
+	}
+
+	.episode_details p {
+		line-height: 4rem;
 	}
 
+	.audioPlayer {
+		grid-area: player;
+		width: 70%;
+		margin: auto;
+		flex-direction: row;
+		align-items: center;
+		align-self: center;
+	}
+
+	.audioPlayerBar {
+		align-items: center;
+		justify-content: flex-end;
+	}
+
+	.audioPlayerPlaybutton {
```

Hi. I think its better to add 
margin-bottom: 0;

```suggestion
	.audioPlayerPlaybutton {
	        gap: 5rem;
		margin-bottom: 0;
	}
```

---

Thanks, @Shiho317 ! I'll update this part.